RepositoryBase.js

```diff
@@ -28,7 +28,7 @@ class RepositoryBase extends PObject {
     /**
      * The model used by this Repository.
      *
-     * @property {puzzleframework.core.db.Model}
+     * @property {sequelize.Model}
```

Comment:
Why did you change it to sequelize.Model?

Reply:
puzzle.models.get returns the sequelize model from the inside of puzzleframework Model:
```
// Model loader class
get(entity) { 
    return this._models[entity]
      ? this._models[entity].model
      : null;
  }
```